### PR TITLE
Sample course: first session: use consistent phrasing #7847

### LINKED ISSUE #7847: Sample course: first session: use consistent phrasing
Questions in the sample sessions are phrased inconsistently, using both 'you' and 'my' to refer to the respondent

Current:
image

Suggestion:
Rephrase the first question as `My estimate of how much each team member contributed`

Even better if we can rephrase all questions to use `You` because it is easier to provide instructions to the respondent when using imperative mood.

### PR DESCRIPTION
**Pull request by amarlearning**
Fixes #7847 

>  the sample questions are created by using data from InstructorSampleData.json and changes will be visible only in new instructor's account.

### CODE REVIEW

**Comment by amarlearning** [comment]
Ready for review

File: src/main/resources/InstructorSampleData.json [hunk 1]
```diff
@@ -516,7 +516,7 @@
       "courseId": "demo.course",
       "creatorEmail": "teammates.demo.instructor@demo.course",
       "questionMetaData": {
-        "value": "{\"distributeToRecipients\":true,\"pointsPerOption\":true,\"questionText\":\"Split points among the your team members and yourself, according to how much you think each member has contributed.\",\"numOfConstSumOptions\":0,\"questionType\":\"CONSTSUM\",\"points\":100,\"constSumOptions\":[]}"
+        "value": "{\"distributeToRecipients\":true,\"pointsPerOption\":true,\"questionText\":\"Split points among your team members and yourself and how much do you think each member has contributed.\",\"numOfConstSumOptions\":0,\"questionType\":\"CONSTSUM\",\"points\":100,\"constSumOptions\":[]}"
```
Comment (whipermr5) [inline src/main/resources/InstructorSampleData.json]: Only the deletion of `the` is necessary; revert the other change.

File: src/main/resources/InstructorSampleData.json [hunk 2]
```diff
@@ -699,7 +699,7 @@
       "courseId": "demo.course",
       "creatorEmail": "teammates.demo.instructor@demo.course",
       "questionMetaData": {
-        "value": "My comments about this teammate(confidential and only shown to instructor)"
+        "value": "Your comments about this teammate(confidential and only shown to instructor)"
```
Comment (whipermr5) [inline src/main/resources/InstructorSampleData.json]: Can take this opportunity to add the missing space before the `(`.

File: src/main/resources/InstructorSampleData.json [hunk 3]
```diff
@@ -743,7 +743,7 @@
       "courseId": "demo.course",
       "creatorEmail": "teammates.demo.instructor@demo.course",
       "questionMetaData": {
-        "value": "My feedback to this teammate(shown anonymously to the teammate)"
+        "value": "Your feedback to this teammate(shown anonymously to the teammate)"
```
Comment (whipermr5) [inline src/main/resources/InstructorSampleData.json]: Same here.

File: src/main/resources/InstructorSampleData.json [hunk 4]
```diff
@@ -824,7 +824,7 @@
       "courseId": "demo.course",
       "creatorEmail": "teammates.demo.instructor@demo.course",
       "questionMetaData": {
-        "value": "My comments about this teammate(confidential and only shown to instructor)"
+        "value": "Your comments about this teammate(confidential and only shown to instructor)"
```
Comment (whipermr5) [inline src/main/resources/InstructorSampleData.json]: Same here.

File: src/main/resources/InstructorSampleData.json [hunk 5]
```diff
@@ -868,7 +868,7 @@
       "courseId": "demo.course",
       "creatorEmail": "teammates.demo.instructor@demo.course",
       "questionMetaData": {
-        "value": "My feedback to this teammate(shown anonymously to the teammate)"
+        "value": "Your feedback to this teammate(shown anonymously to the teammate)"
```
Comment (whipermr5) [inline src/main/resources/InstructorSampleData.json]: Same here.

File: src/main/resources/feedbackSessionTeamEvaluationTemplate.json [hunk 6]
```diff
@@ -4,7 +4,7 @@
     "courseId": "${courseId}",
     "creatorEmail": "${creatorEmail}",
     "questionMetaData": {
-      "value": "{\"questionText\":\"My estimate of how much each team member contributed.\",\"questionType\":\"CONTRIB\"}"
+      "value": "{\"questionText\":\"According to you how much each team member has contributed.\",\"questionType\":\"CONTRIB\"}"
```
Comment (whipermr5) [inline src/main/resources/feedbackSessionTeamEvaluationTemplate.json]: Changing `My` to `Your` in the original would suffice.

File: src/main/resources/feedbackSessionTeamEvaluationTemplate.json [hunk 7]
```diff
@@ -30,7 +30,7 @@
     "courseId": "${courseId}",
     "creatorEmail": "${creatorEmail}",
     "questionMetaData": {
-      "value": "Comments about my contribution(shown to other teammates)"
+      "value": "Comments about your contribution(shown to other teammates)"
```
Comment (whipermr5) [inline src/main/resources/feedbackSessionTeamEvaluationTemplate.json]: Can add the missing space here too.

File: src/main/resources/feedbackSessionTeamEvaluationTemplate.json [hunk 8]
```diff
@@ -61,7 +61,7 @@
     "courseId": "${courseId}",
     "creatorEmail": "${creatorEmail}",
     "questionMetaData": {
-      "value": "My comments about this teammate(confidential and only shown to instructor)"
+      "value": "Your comments about this teammate(confidential and only shown to instructor)"
```
Comment (whipermr5) [inline src/main/resources/feedbackSessionTeamEvaluationTemplate.json]: Same here.

File: src/main/resources/feedbackSessionTeamEvaluationTemplate.json [hunk 9]
```diff
@@ -105,7 +105,7 @@
     "courseId": "${courseId}",
     "creatorEmail": "${creatorEmail}",
     "questionMetaData": {
-      "value": "My feedback to this teammate(shown anonymously to the teammate)"
+      "value": "Your feedback to this teammate(shown anonymously to the teammate)"
```
Comment (whipermr5) [inline src/main/resources/feedbackSessionTeamEvaluationTemplate.json]: Same here.

**Comment by amarlearning** [comment]
ready for review

**Comment by amarlearning** [comment]
@whipermr5 reminder

**Pull request merged**